docs-en/14-reference/05-taosbenchmark.md

```diff
@@ -35,17 +35,17 @@ Execute the following commands to quickly experience taosBenchmark's default con
 taosBenchmark
 ```
 
-When run without parameters, taosBenchmark connects to the TDengine cluster specified in `/etc/taos` by default and creates a database named test in TDengine, a super table named `meters` under the test database, and 10,000 tables under the super table with 10,000 records written to each table. Note that if there is already a test database, this table is not used. Note that if there is already a test database, this command will delete it first and create a new test database.
+When run without parameters, taosBenchmark connects to the TDengine cluster specified in `/etc/taos` by default and creates a database named test in TDengine, a super table named `meters` under the test database, and 10,000 tables under the super table with 10,000 records written to each table. Note that if there is already a test database, this command will delete it first and create a new test database.
 
 ### Run with command-line configuration parameters
 
-The `-f <json file>` argument cannot be used when running taosBenchmark with command-line parameters and controlling its behavior. Users must specify all configuration parameters from the command-line. The following is an example of testing taosBenchmark writing performance using the command-line approach.
+The `-f <json file>` argument cannot be used when running taosBenchmark with command-line parameters. Users must specify all configuration parameters from the command-line. The following is an example of testing taosBenchmark writing performance using the command-line approach.
 
 ```bash
 taosBenchmark -I stmt -n 200 -t 100
 ```
 
-The above command, `taosBenchmark` will create a database named `test`, create a super table `meters` in it, create 100 sub-tables in the super table and insert 200 records for each sub-table using parameter binding.
+In the above command, `taosBenchmark` will create the default database named `test`, create the default super table named `meters`, create 100 subtables in the super table and insert 200 records for each subtable.
```

Comment:
'-I stmt' means explicitly specifying the "parameter binding" interface which is no taos_query() series interface. Please include the difference.

Reply:
Interesting, I have never heard of parameter binding interface writing. Parameter binding is typical for front end interfaces, as in modern web design, but I have never heard of writing to a database in this way. Is there another software package that uses this "parameter binding" interface that you can point me to?

Reply:
Please see the link https://docs.tdengine.com/develop/insert-data/sql-writing#insert-with-parameter-binding. Actually, MySQL supports this style interface first. TDengine implements a set of similar interfaces.